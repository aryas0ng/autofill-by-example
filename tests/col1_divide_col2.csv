1,0,0
2,4,0.5
2,3,0.666666666666667
3,8,0.375
10,10,1
5,5,
4,4,
19,4,
2,21,
1,21,
9,10,
7,7,
4,4,
3,3,
12,3,
4,16,
1,16,
26,27,
8,8,
3,3,
3,3,
8,11,
4,4,