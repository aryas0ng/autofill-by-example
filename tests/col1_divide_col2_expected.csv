1,0,0
2,4,0.5
2,3,0.666666666666667
3,8,0.375
10,10,1
5,5,1
4,4,1
19,4,4.75
2,21,0.0952380952380952
1,21,0.0476190476190476
9,10,0.9
7,7,1
4,4,1
3,3,1
12,3,4
4,16,0.25
1,16,0.0625
26,27,0.962962962962963
8,8,1
3,3,1
3,3,1
8,11,0.727272727272727
4,4,1